README.md

```diff
@@ -7,6 +7,7 @@ Example of creating and deploying an API with Docker and Terraform on AWS.
 * Install the [awslogin](https://pypi.org/project/byu-awslogin/) CLI tool
 * Log into a dev account (with awslogin)
 * Ensure your account has a [Terraform State S3 Backend](https://github.com/byu-oit/terraform-aws-backend-s3) deployed.
+* Obtain DivvyCloud Username and Password from Cloud Office - cloudoffice@byu.edu. Only needed if deploying outside the byu-oit Github Organization.
```

Comment:
nit: I'd probably put the not about only needing it outside of `byu-oit` before the step so people who don't read the whole thing spend time getting a password they don't need.

Reply:
See #107. 🙂